libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx

```diff
@@ -40,6 +40,7 @@ export default function ImportantNotes(props: IImportantNotesProps) {
           }}
         >
           <Textarea
+            textAlignVertical='top'
```

Comment:
Maybe we could we set this on all `Textarea` components as a default for consistency?
As long as we could still override it when needed.